### ISSUE TITLE
[TF2] [VScript] Permit loading two workshop addons at once

### ISSUE THREAD

**Issue by KaelaSavia** [ValveSoftware/Source-1-Games#5550]
TF2 currently has limitation that each custom vscript gamemode needs to have its content and vscript files packed into each map that uses it.
- And if such gamemode vscript needs update, then all maps need to be recompiled/repacked and redownloaded

This makes maintaining custom gamemodes in workshop with more than few maps quite infeasible and even more infeasible if someone would like to make fan map for such custom gamemode. 

Solution could be ability for workshop map to be able to mark another workshop map as dependency and load its packed content into memory at same time as main map is played.

Basically a dummy .bsp file containing vscript gamemode's code/models/materials/sounds that is loaded along with main map .bsp file
This would make maintaining and updating vscript content more feasible

If you have 50+ surf maps with surf vscript, you would only need to update 1 workshop addon that is mounted by each map. And other map makers wouldn't need to do it either (Gmod/CS2 style).

It's quite a big feature to be requested but as seen by how vscript ZI or VSH maintaining is affected by current limitations, I believe it would be worth it!
